src/sorting/python/heap_sort/tests/test_heap_sort.py

```diff
@@ -6,39 +7,49 @@
 file_dir = os.path.dirname(os.path.dirname(__file__))
 sys.path.append(file_dir + "/src")
 
```

Comment:
As I speak to Evan I see all this as not really serious things, and as this is a test task for learning 
I also don't find it nessesary to write the docstrings for tests, just i make them because of guides
Let's just move on )

Just we will ask about items assertions to know better